CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.1.2
+
+- Handle superfluous `version` keys with `git` and `path` dependencies.
+- Improve errors when unsupported keys are provided in dependencies.
+- Provide better errors with invalid `sdk` dependency values.
+- Support "scp-like syntax" for Git SSH URIs.
```

Comment:
could we have more explanation or a link for what "scp-like syntax" means? Could copy the example you have in the doc comment: `[user@]host.xz:path/to/repo.git/`

Reply:
ack